reporter.go

```diff
@@ -234,7 +235,7 @@ func (r *remoteReporter) processQueue() {
 				atomic.AddInt64(&r.queueLength, -1)
 				if flushed, err := r.sender.Append(span); err != nil {
 					r.metrics.ReporterFailure.Inc(int64(flushed))
-					r.logger.Error(err.Error())
+					r.logger.Error(fmt.Sprintf("span %q: %s", err.Error()))
```

Comment:
this is not actually adding any additional info about the span, only the error is passed in. Why `%q` format?

👍  on better error message prefix. 

👎 on dumping the whole span (especially given the actual situation that led to this PR).

Reply:
oops my bad, forgot to include operationname.. amended the commit. please check again.
`%q` format is because people might put arbitrary characters into the operation name (e.g. whitespace)